sap_host_exporter@.service

```diff
@@ -1,11 +1,12 @@
 [Unit]
 Description=Prometheus exporter for Netweaver clusters metrics
+Documentation=https://github.com/SUSE/sap_host_exporter
 After=network.target
 
 [Service]
 Type=simple
 Restart=always
-ExecStart=/usr/bin/sap_host_exporter
+ExecStart=/usr/bin/sap_host_exporter  --identifier %i
```

Comment:
hmm, while I think I can imagine what this would do, at the moment we don't have an `--identifier` flag, so this would break the current unit file

Reply:
ah yeah right I was thinking was from spec thing specific :grin: